src/test/java/com/mentoring/pageobject/Page.java

```diff
@@ -62,4 +62,12 @@ boolean isElementDisplayed(WebElement element) {
 
     public abstract void waitUntilPageLoads();
 
+    void sleepForSeconds(int sec) {
+        try {
+            Thread.sleep(1000 * sec);
+        } catch (InterruptedException e) {
+            log.error(e.getStackTrace().toString());
```

Comment:
You don't need to get the stack trace such way. With Slf4j logging you can say: log.error("Custom error message", e). It's good practice to add custom messages for exceptions that helps to identify the failure much faster (here e.g: "Waiting with Thread.sleep failed").